### DISCUSSION TITLE
Filter for invalid feeds

### DISCUSSION THREAD

**Discussion by bsdooby** [ViennaRSS/vienna-rss#1338] [Ideas]
It should be possible (best way to think of this is as a smart list) to have invalid feeds show up at a central location, in an easily parseable log window, etc.

As RSS is under pressure, a lot of once great blogs's  feeds are no longer valid, and it is hard to hunt them down in the current implementation of Vienna. Further, RSS feed enpoints might change, and spotting these changes increases the usability of Vienna considerably.

**Comment by barijaona** [comment]
Did you notice that you can invoke the Activity window (shortcut : Command-0), sort its content by status, and directly obtain from there additional info for the selected feed (shortcut : Command-i) ?

**Comment by bsdooby** [comment]
Didn't know about that. That's nice. Thank you for that hint.
IMHO, it would be more convenient to have this exact feature as a smart
folder/list, or a filterable feature.
